<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adduser_Dialog</class>
 <widget class="QDialog" name="adduser_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Создание пользователя</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>61</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Имя пользователя</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>102</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>142</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль(подтверждение)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_lineEdit">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>57</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd_lineEdit">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>97</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd2_lineEdit">
   <property name="geometry">
    <rect>
     <x>218</x>
     <y>137</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="accept_pushButton">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>224</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Принять</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>224</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
